src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -181,6 +199,7 @@ const GeneExternalReferences = () => {
 
 const TranscriptExternalReferencesGroups = (props: {
   transcript: Transcript;
+  externalReferenceClickHandler: (linkLabel: string) => void;
```

Comment:
`onClick`? `onReferenceClick`? From the viewpoint of this component, there's only one thing that one can click.

Reply:
✅